### DISCUSSION TITLE
How can I keep a minimized window active?

### DISCUSSION THREAD

**Discussion by Jerenyaoyelu** [tauri-apps/tauri#11948] [Q&A]
I need to keep timer active after the window is minimized or hidden（not display in front）, how can I do it? something like backgroundThrottling:false in electron

**Comment by FabianLars** [comment]
This is not possible, at least not yet (though ii rc the webviews don't allow us to mirror electron's config but not sure right now).
For now you'll have to do that in a web worker or on the rust side.

**Reply by Jerenyaoyelu** [reply]
This approach may not be suitable for my use case, because I am creating a sub window and inject some js code, then the sub window needs to be invisible or hidden or minimized. If I understood correctly, the web worker in rust side would not affect the sub window, right?
